package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "webpack-cli",
+  "name": "webpack-cli-monorepo",
```

Comment:
Keep it the same as before.
We can follow babel here https://github.com/babel/babel/blob/master/package.json

Reply:
This is just a placeholder name. This won't be published to `npm`. In fact I am going to make a further change, I forgot to add `"private": true` to the `package.json`